### DISCUSSION TITLE
osu wont open without admin

### DISCUSSION THREAD

**Discussion by Itsjustmelandon** [ppy/osu#20976] [Q&A]
### Type

Performance

### Bug description

osu!lazer wont open without administrator access but as soon as i try administrator it starts working again

### Screenshots or videos

![osu_2022-10-27_17-29-12](https://user-images.githubusercontent.com/65632159/198421910-d53bee94-9227-437b-89e2-e04e06aebccf.png)

### Version

2022.1022.0

### Logs

[database.log](https://github.com/ppy/osu/files/9883824/database.log)
[input.log](https://github.com/ppy/osu/files/9883825/input.log)
[network.log](https://github.com/ppy/osu/files/9883826/network.log)
[performance.log](https://github.com/ppy/osu/files/9883827/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9883828/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9883829/updater.log)

**Comment by peppy** [comment]
Could you please provide logs after attempting to open as a normal user?

**Comment by Itsjustmelandon** [comment]
> Could you please provide logs after attempting to open as a normal user?

sorry yes i will
[database.log](https://github.com/ppy/osu/files/9884771/database.log)
[input.log](https://github.com/ppy/osu/files/9884772/input.log)
[network.log](https://github.com/ppy/osu/files/9884773/network.log)
[performance.log](https://github.com/ppy/osu/files/9884774/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9884775/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9884776/updater.log)
i think nothing happened but i have no knowledge of how this works

**Comment by peppy** [comment]
Not sure. You might have to check event viewer. There's nothing in logs to suggest osu! even started.

**Comment by Ru00RU** [comment]
It has something to do with app NTFS permission on his PC. which didn't allow OSU lazer to access some file

I once have issue like this. My case was I log in my window with organization account and it apply some policies on to my PC.
This is how I fix it (The troubleshooting result come from trial and error)

open Command prompt on Administration mode (right click on it and "run as administrator"
run the NTFS permission reset command (without quotation) by input "ICACLS * /T /Q /C /RESET" then press enter
this should reset your PC permission to default state. (don't forget asterisk)

to be sure I also run another command by input "icacls C:\Users\USERNAME\AppData /RESET" to reset AppData file permission as well (I don't know if this was needed)

USERNAME should be your user name that has AppData in it.



This instruction should help fix his problem.

(sorry if my English wasn't the best)
